dev/src/cli/cli_create.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path';
+import { isCancel, select, text } from '@clack/prompts';
+import { exec, execSync } from 'child_process';
+import { promisify } from 'util';
```

Comment:
for node related imports can you please add the `node:` prefix? Thanks!

like: `import * as path from 'node:path';`